sorc/app_build.sh

```diff
@@ -202,9 +202,13 @@ EXTERNALS="${EXTERNALS^^}"
 
 # check if PLATFORM is set
 if [ -z $PLATFORM ] ; then
-  printf "\nERROR: Please set PLATFORM.\n\n"
-  usage
-  exit 0
+  # Automatically detect HPC platforms for wcoss2, hera, jet, orion, etc
+  source ../ush/fix_rrfs_locations.sh
```

Comment:
@guoqing-noaa, can you change this relative path to the absolute path like: `source ${HOME_DIR}/ush/fix_rrfs_locations.sh`?

Reply:
Done. Thanks!